packages/crackql/crackql.install

```diff
@@ -0,0 +1,15 @@
+post_install() {
+  set -e
+  cd /usr/share/crackql
+  python -m virtualenv venv
+  source venv/bin/activate &&
+    pip install --isolated --root="/usr/share/crackql" --prefix='venv' \
+      'certifi' 'charset-normalizer' 'graphql-core' 'idna' 'jinja2' \
```

Comment:
Why don't you pull them from https://github.com/nicholasaleks/CrackQL/blob/master/requirements.txt as per https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/python-standalone.install?

Reply:
Packages and run fine, however I do know get this error:
ERROR: file:///usr/share/crackql does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.

~!>D

Reply:
I can make a request to the devs to create proper project.toml file. Let me know.

Reply:
fyi. I talked to the devs. They will add a setup file. 
Also his new book coming out --
https://nostarch.com/black-hat-graphql

That's awesome.

Reply:
https://github.com/nicholasaleks/CrackQL/issues/8